src/common/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -219,6 +220,7 @@ bool ov::pass::CommonOptimizations::run_on_model(const std::shared_ptr<ov::Model
     REGISTER_PASS(manager, ConvertInterpolate11ToInterpolate4)
     REGISTER_PASS(manager, ConvertPad12ToPad1)
     REGISTER_PASS(manager, ConvertScatterElementsUpdate12ToScatterElementsUpdate3)
+    REGISTER_PASS(manager, ConcatFusion)
```

Comment:
Should we move it to MOC transformations? 
Also it would be better to combine it with other "Fuse" transformations

Reply:
Yeah, I guess it can be a part of MOC